core/src/test/scala/org/apache/spark/ui/UIUtilsSuite.scala

```diff
@@ -122,8 +122,8 @@ class UIUtilsSuite extends SparkFunSuite {
   test("decodeURLParameter (SPARK-12708: Sorting task error in Stages Page when yarn mode.)") {
     val encoded1 = "%252F"
     val decoded1 = "/"
-    val encoded2 = "%253Cdriver%253E"
-    val decoded2 = "<driver>"
+    val encoded2 = "driver"
```

Comment:
You can delete this check I think; there is nothing to encode

Reply:
done. 
Removed unnecessary encode.